diagnostic.md

```diff
@@ -70,15 +62,11 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
 
- <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+Absolute because the use of the command doesnt start from the root.
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
- <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+Absolute because you are already in the directory.
```

Comment:
Relative, the image file lives inside our current working directory, project ; as a result, an absolute path is not needed to access it.